src/overlays/actors/ovl_Bg_Icefloe/z_bg_icefloe.c

```diff
@@ -27,35 +34,148 @@ const ActorInit Bg_Icefloe_InitVars = {
     (ActorFunc)BgIcefloe_Update,
     (ActorFunc)BgIcefloe_Draw,
 };
+static BgIcefloe* D_80AC4F30[] = { NULL, NULL, NULL };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80AC4F3C[] = {
+static InitChainEntry sInitChain[] = {
     ICHAIN_VEC3F_DIV1000(scale, 0, ICHAIN_STOP),
 };
 
-#endif
-
-extern InitChainEntry D_80AC4F3C[];
-
-extern UNK_TYPE D_060001E0;
-extern UNK_TYPE D_06000C90;
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Bg_Icefloe/BgIcefloe_Init.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Bg_Icefloe/BgIcefloe_Destroy.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Bg_Icefloe/func_80AC4A80.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Bg_Icefloe/func_80AC4AE8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Bg_Icefloe/func_80AC4C18.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Bg_Icefloe/func_80AC4C34.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Bg_Icefloe/func_80AC4CF0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Bg_Icefloe/func_80AC4D2C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Bg_Icefloe/BgIcefloe_Update.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Bg_Icefloe/BgIcefloe_Draw.s")
+static s32 D_80AC5060;
```

Comment:
```suggestion
s32 D_80AC5060;
```
For bss, we try our best to keep them non static. If you are curious why feel free to ask in the discord.